### DISCUSSION TITLE
"Status" on npmjs.com for Nuxt VueFire is "in Preview"

### DISCUSSION THREAD

**Discussion by mgav** [vuejs/vuefire#1522] [Questions and Help] [answered]
Aloha. Status for Nuxt Vuefire at [https://www.npmjs.com/package/vuefire](https://www.npmjs.com/package/vuefire) is "in Preview" but "Status" [here](https://github.com/vuejs/vuefire) is that "VueFire and Nuxt VueFire are both currently stable" I'm not sure which is correct or how to fix the incorrect one. I'm also not 100% clear on whether it's only Nuxt 2, or also Nuxt 3. Thanks!

**Comment by posva** [comment, marked as answer]
The up to date version is the readme here and the docs 🙂
